ha/core/controllers/pcs/node_controller.py

```diff
@@ -152,6 +149,119 @@ def status(self, nodeids: list = None) -> dict:
         """
         raise HAUnimplemented("This operation is not implemented.")
 
+    def _validate_permissions(self) -> None:
+
+        # confirm that user is root or part of haclient group"
+        user = getpass.getuser()
+        group_id = os.getgid()
+
+        try:
+            # find group id for root and haclient
+            id_ha = grp.getgrnam(const.USER_GROUP_HACLIENT)
+            id_root = grp.getgrnam(const.USER_GROUP_ROOT)
+
+            # if group not root or haclient return error
+            if group_id != id_ha.gr_gid and group_id != id_root.gr_gid:
+                Log.error(f"User {user} does not have necessary permissions to execute this CLI")
+                raise HAInvalidPermission(
+                            f"User {user} does not have necessary permissions to execute this CLI")
+
+            # The user name "hauser"; which is part of the "haclient" group;
```

Comment:
Is this comment needed now ? If not remove it

Reply:
No, I removed this comment.

Reply:
I still see all the commented lines